devices/Hx711/Hx711.cs

```diff
@@ -173,12 +203,24 @@ private void SetChannelAndGainFactor()
         {
             // send N clock pulses according to the set gain factor
             // 1 clock pulse per gain factor
-            _spiDevice.WriteByte((byte)Gain);
+            Debug.WriteLine("INFO: Setting channel and gain.");
+
+            // setup buffer to drive PD_SCK
+            SpanByte clkTrain = new(_readSamplePulseTrain);
```

Comment:
This one here and the other readBuffer should be declared in the constructor to improve efficiency. 
No need to allocate every time a read is performed.

Reply:
I need some hint how to move SpanByte to constructor

Reply:
You need to declare it as a class property and init in constructor.
Take a look at `_readSamplePulseTrain`.

Reply:
Thanks for pointing me to weak places. I still keep creating 1 byte long buffers for DOUT signal waiting. I that is making OS to do huge work (as this is only function that is always active) then I can move them to constructor too.

Reply:
Note that this is nothing that the CLR can't handle! It's just for efficiency sake. As the buffers are being reused over and over, no point on creating those again and again when it's perfectly fine to reuse them.

It would be critical if this was a high speed code path, but it's not, so all good with _just_ moving the buffers to the class.